### ISSUE TITLE
Updating email in Account doesn't update profile

### ISSUE THREAD

**Issue by klecuyer** [gctools-outilsgc/concierge#79]
As a user, when I need to update my email address in Account, it reflects in my Account but not on my profile. This is after logging out and logging back in with updated email.

This user changed her email to: mary.o'shea@pco-bcp.gc.ca
Old email was mary.oshea@queensu.ca

I deleted her duplicate accounts (this may be what caused it). In her account her email is now up to date reflecting the new email:
![image](https://user-images.githubusercontent.com/29754185/44164160-4569d300-a093-11e8-888b-8cad39975e43.png)

Even after logging in with the new email (@pco) her profile still reflects the old email: https://gccollab.ca/profile/Mary.Oshea

![image](https://user-images.githubusercontent.com/29754185/44164223-6af6dc80-a093-11e8-9dc2-6f19a171579b.png)

Refreshing the page and logging in/out does not update the email address.

Freshdesk ticket: https://gccollab.gctools-outilsgc.ca/helpdesk/tickets/4686

**Comment by klecuyer** [comment]
@marianneaubrey15 @bryan-robitaille

**Comment by klecuyer** [comment]
Looking at the profile from my admin account it looks like the email is up to date and has been changed:

![image](https://user-images.githubusercontent.com/29754185/44164550-59620480-a094-11e8-80b2-82e6bf50827f.png)

But I just logged in as the user in a new session and it still shows her old Queens email (not updated).

![image](https://user-images.githubusercontent.com/29754185/44164649-9e863680-a094-11e8-9282-0e83b0f03ca8.png)

To summarize, the user doesn't see the changes themselves but others do.

**Comment by klecuyer** [comment]
Also updating the display name doesn't change in profile: https://gccollab.ca/profile/Lara.Arsenault

Trying to fix misspell of 'Disaster' in display name

Freshdesk ticket: https://gccollab.gctools-outilsgc.ca/helpdesk/tickets/4742

**Comment by klecuyer** [comment]
Group information (About) doesn't update or delays, following testing with @bryan-robitaille here: https://gccollab.ca/groups/profile/1083031/entesting-descriptionsfr

(kubernetes migration causing delay between front/backend). Refreshing the page causes inconsistent results.
